style.css

```diff
@@ -43,4 +43,27 @@ button:active {
 
 .item button {
   margin-left: 10px;
+}
+
+.sum {
+  display: flex;
+  align-items: center;
+}
+
+.number {
+  display: flex;
+  align-items: center;
+}
+
+.number button {
+  margin-left: 20px;
+}
+
+.choose {
+  display: flex;
+  align-items: center;
+}
+
+.choose button {
+  margin-left: 20px;
```

Comment:
Why do you need it?

Reply:
I agree, this is unnecessary, I will fix it